R/truncnorm.R

```diff
@@ -31,6 +33,9 @@ my_etruncnorm= function(a,b,mean=0,sd=1){
   toobig = max_alphabeta<(-30)
   toobig[is.na(toobig)]=FALSE
   tmp[toobig] = max_ab[toobig]
+
+  sdzero = sd == 0
+  tmp[sdzero] = mean[sdzero]
```

Comment:
I thought about that too. In that case the distribution is not well defined. I did not come up with a solution.